youtube_dl/extractor/openload.py

```diff
@@ -254,7 +254,15 @@ class OpenloadIE(InfoExtractor):
                         (?:f|embed)/
                         (?P<id>[a-zA-Z0-9-_]+)
                     ''' % _DOMAINS
+    _EMBED_WORD = 'embed'
+    _STREAM_WORD = 'f'
+    _REDIR_URL = 'stream'
+    _URL_IDS = ('streamurl', 'streamuri', 'streamurj')
 
+    def _extract_decrypted_page(self, page_url, webpage, video_id, headers):
```

Comment:
incorrect order, put the methods after the properties.

Reply:
Done